### DISCUSSION TITLE
ImportFile is returning only 50 importFiles

### DISCUSSION THREAD

**Discussion by vsridharan123** [Accenture/sfmc-devtools#482] [Q&A] [answered]
When I'm running the below command
"mcdev r EnterpriseDigital/AHM_Testing importFile" or "mcdev r EnterpriseDigital/AHM_Testing", it is returning only 50 import files
eg: on the last two lines of the output:

13:25:11 info: Downloaded: importFile (50)
13:25:11 info: :: Done

In total, there are 125+ import activities.

Please advise.

**Comment by JoernBerkefeld** [comment]
hi Vinod, `importFile` is one of the types that have been troubling us for quite some time. The APIs don't seem to return all entries for some reason and so far we have not found out why. You can confirm that by calling SFMC apis via Postman.

The typical recommendation is to list importFile activities as a pre-deployment step. At this point we don't know what else to try here :( 


related bug ticket: #198

**Reply by JoernBerkefeld** [reply, marked as answer]
the limit of 50 records was fixed with release [5.0.0](https://github.com/Accenture/sfmc-devtools/releases/tag/v5.0.0)
